Lib/test/support/__init__.py

```diff
@@ -945,6 +945,27 @@ def check_sizeof(test, o, size):
             % (type(o), result, size)
     test.assertEqual(result, size, msg)
 
+def subTests(arg_names, arg_values, /, *, _do_cleanups=False):
+    """Run multiple subtests with different parameters.
+    """
+    if isinstance(arg_names, str):
+        arg_names = arg_names.split(',')
```

Comment:
Could this also `strip()` the names? We should allow putting space after a comma.